src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -255,23 +255,12 @@ static void registerBukkitEvents() {
 			if (e.equals(PlayerInteractAtEntityEvent.class) || e.equals(PlayerArmorStandManipulateEvent.class)) {
 				continue; // Ignore, registered above
 			}
-			
-			if (!containsSuperclass((Set) registeredEvents, e)) { // I just love Java's generics
+			if (!registeredEvents.contains(e)) { // Check if event is registered
```

Comment:
If this works properly, it's fine with me for now. In the future, a better system where we actually unregister listeners would be good. Can't happen until after https://github.com/SkriptLang/Skript/pull/4108 though :)